src/api/eth.rs

```diff
@@ -416,6 +418,7 @@ mod tests {
     "cumulativeGasUsed": "0x33bc",
     "gasUsed": "0x4dc",
     "contractAddress": "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
```

Comment:
Your `EXAMPLE_RECEIPT` states that it was taken from the Ethereum JSON-RPC docs but did not include `logsBloom`. I verified the docs: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionreceipt ; I do see a value for `logsBloom`, so I included it. I'm not sure if there was a specific reason for excluding it. All tests are now passing on my side.